Second Assignment

panhandleprep opened the pull request:
Here is my second assignment. I basically gathered all of the info from exercises one, two and three and changed or added parts to the code. I'm not 100% about my "if... else statements" I mean they work and everything but I guess I'm just wondering how i could have changed them if i wanted to make a more complex organized pattern that wasn't so random.

panhandleprep committed:
Second Assignment 

Here is my second assignment. I basically gathered all of the info from exercises one, two and three and changed or added parts to the code. I'm not 100% about my "if... else statements" I mean they work and everything but I guess I'm just wondering how i could have changed them if i wanted to make a more complex organized pattern that wasn't so random.

andyinabox commented:
Great work! One silly mistake -- it looks like you pasted the contents of `index.html` into `README.md` by accident!

Also, it looks like you were probably working in the `example-basic` folder, which is fine. Just make sure when you paste into the main `index.html` file you replace this:

```
        <script language="javascript" type="text/javascript" src="../p5/p5.min.js"></script>
        <script language="javascript" type="text/javascript" src="../p5/addons/p5.dom.js"></script>
```

with

```
        <script language="javascript" type="text/javascript" src="p5/p5.min.js"></script>
        <script language="javascript" type="text/javascript" src="p5/addons/p5.dom.js"></script>
```

Or the p5 JavaScript won't load correctly.

panhandleprep committed:
Assignment 2

panhandleprep committed:
Update README.md

panhandleprep committed:
Exercise 4 

so here is my code I could not get it to show up i looked at the console over and over again and it says that i made a mistake on line 114 but when i went to check it seemed correct to me

panhandleprep committed:
Exersise

panhandleprep committed:
Exercise 4 (resubmit)

here is my resubmission for exercise I realize what i did wrong and I also added some hovers to some of that characters
